src/e3/anod/spec.py

```diff
@@ -78,6 +78,28 @@ def has_primitive(anod_instance: Anod, name: str) -> bool:
     return is_primitive
 
 
+def fetch_attr(instance: Any, name: str, default_value: Any) -> Any:
+    """Return an attribute or the default value if missing.
+
+    Unlike `getattr(instance, name, default_value)`, this works only on
+    attributes present in the class (so class attributes or properties) and
+    this does not hide AttributeError exceptions that getting an existing
+    attribute might raise.
+    """
+
```

Comment:
no blank line here, this has been detected by the CI:

`src/e3/anod/spec.py:83:1: D202 No blank lines allowed after function docstring
1027`

You can run it locally with `tox -e checkstyle`

Reply:
Done, thanks. I proably missed this one because I can’t get a clean `tox` run even with the current `master` branch. FTR, I get this:

```
>                       raise DistributionNotFound(req, requirers)
E                       pkg_resources.DistributionNotFound: The 'netifaces' distribution was not found and is required by the application

/home/pmderodat/_git/e3-core/.tox/py38-cov-xdist/lib/python3.8/site-packages/pkg_resources/__init__.py:770: DistributionNotFound
```
Please let me know if it’s worth opening an issue.

Reply:
As discussed live this was a setup issue on your side and you are now able to run tox.